src/home/EigenTVLOvertimeChart.jsx

```diff
@@ -0,0 +1,232 @@
+import { AxisBottom, AxisLeft } from '@visx/axis';
+import { localPoint } from '@visx/event';
+import { GridColumns, GridRows } from '@visx/grid';
+import { Group } from '@visx/group';
+import { LegendOrdinal } from '@visx/legend';
+import { scaleLinear, scaleOrdinal, scaleTime } from '@visx/scale';
+import { Circle, LinePath } from '@visx/shape';
+import { TooltipWithBounds, useTooltip } from '@visx/tooltip';
+import { extent, max } from 'd3-array';
+import { useCallback, useMemo } from 'react';
+import { formatDateToVerboseString, formatNumber } from '../utils';
+
+const getNumberOfTicks = (width, axis) => {
+  if (axis === 'x') {
+    if (width < 500) return 2;
+    if (width < 800) return 5;
+    return 7;
+  } else if (axis === 'y') {
+    if (width < 500) return 3;
+    if (width < 800) return 4;
+    return 5;
+  }
+};
+
+const EigenTVLOvertimeChart = ({ data, width, height }) => {
+  const {
+    tooltipData,
+    tooltipLeft = 0,
+    tooltipTop = 0,
+    showTooltip,
+    hideTooltip
+  } = useTooltip();
+
+  const margin = { top: 40, right: 0, bottom: 40, left: 45 };
+
+  const innerWidth = width - margin.left - margin.right;
+  const innerHeight = height - margin.top - margin.bottom;
+
+  const getEthTVL = d => parseFloat(d.ethTVL) / 1e18;
+  const getLstTVL = d => parseFloat(d.lstTVL) / 1e18;
+
+  const getTvlByDate = date => {
+    const selectedDate = formatDateToVerboseString(date, 'yyyy-MM-dd');
+    const tvl = data.find(
+      item =>
+        formatDateToVerboseString(new Date(item.timestamp), 'yyyy-MM-dd') ===
+        selectedDate
+    );
+    return tvl;
+  };
+
+  const dateScale = useMemo(
+    () =>
+      scaleTime({
+        range: [0, innerWidth],
+        domain: extent(data, d => new Date(d.timestamp)),
+        nice: true
+      }),
+    [data, innerWidth]
+  );
+
+  const tvlScale = useMemo(
+    () =>
+      scaleLinear({
+        range: [innerHeight, 0],
+        domain: [0, max(data, d => Math.max(getEthTVL(d), getLstTVL(d)))],
+        nice: true
+      }),
+    [data, innerHeight]
+  );
+
+  const handleTooltip = useCallback(
+    event => {
+      const { x, y } = localPoint(event) || { x: 0, y: 0 };
+      const date = dateScale.invert(x - margin.left);
+
+      showTooltip({
+        tooltipData: getTvlByDate(date),
+        tooltipLeft: x,
+        tooltipTop: y
+      });
+    },
+    [dateScale, margin.left, showTooltip, getTvlByDate]
```

Comment:
need to wrap getTvlByDate in useCallback, or move it into handleTooltip . Same for `getEthTVL` and `getLstTVL`
```suggestion
    [dateScale, margin.left, showTooltip]
```